AlphaWallet/Tokens/ViewControllers/TokenInstanceViewController.swift

```diff
@@ -121,10 +122,16 @@ class TokenInstanceViewController: UIViewController, TokenVerifiableStatusViewCo
 
         header.configure(viewModel: .init(tokenObject: tokenObject, server: tokenObject.server, assetDefinitionStore: assetDefinitionStore))
 
-        let actions = viewModel.actions
-        buttonsBar.numberOfButtons = actions.count
-        buttonsBar.configure()
-        for (action, button) in zip(actions, buttonsBar.buttons) {
+        buttonsBar.configure(.combined(buttons: viewModel.actions.count))
+        buttonsBar.delegate = self
+        buttonsBar.dataSource = self
+
+        var actions = viewModel.actions
+        actions.removeFirst(buttonsBar.buttons.count)
+        moreActions = actions
```

Comment:
- [ ] This doesn't look so good and makes ButtonsBar leaky, i.e. the user (code) has to maintain `moreActions`. Can we try this approach instead:

In here, configure a button for every action, including those that will end up in the "more" button (I think it is already done this way):

```
for (action, button) in zip(viewModel.actions, buttonsBar.buttons) {
```

Then can the actionsheet which is managed by `ButtonsBar` look into these buttons and examine them for the target + action pairs (using `UIButton.allTargets` and `UIButton.actions(forTarget:forControlEvent:`), and fire those? Seems like this can get rid of the dataSource and delegate (I know I suggested to have them) entirely.

Reply:
@hboon So you suggest to store all buttons in `buttons`, show only 2, and rest of buttons mark 
 as `asHidden=true`, and examine them for title and action, to show `ActionSheet`, right?

Reply:
@vladyslav-iosdev yes. But do point out if that wouldn't work. It just came to my mind when I saw `moreActions` and thought of how to improve it.

Reply:
@hboon i guess it should work. i will check it now.